### DISCUSSION TITLE
VOC - Ultralytics YOLOv8 Docs

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#5376] [Docs]
# VOC - Ultralytics YOLOv8 Docs

A complete guide to the PASCAL VOC dataset used for object detection, segmentation and classification tasks with relevance to YOLO model training.

https://docs.ultralytics.com/datasets/detect/voc/

**Comment by RenyooZhang** [comment]
May I ask how should I organize the dataset dictionary? Just the same as what it is in VOC dataset zip?

**Reply by pderrenger** [reply]
@RenyooZhang certainly! When organizing your dataset for training with YOLOv8, you should follow the standard directory structure expected by the model. For a VOC-like dataset, the structure typically looks like this:

```
/parent_folder
    /dataset
        /images
            /train  # training images go here
            /val    # validation images go here
        /labels
            /train  # training labels go here
            /val    # validation labels go here
```

Each image in the `/images/train` and `/images/val` directories should have a corresponding label file in the `/labels/train` and `/labels/val` directories, respectively. The label files should be in a plain text format, with one row per object, and each row should contain class x_center y_center width height formatted as normalized coordinates (i.e., values between 0 and 1 relative to image width and height).

Make sure your dataset configuration file (usually a `.yaml` file) reflects the correct paths to your images and labels. If you need further guidance on the dataset configuration, please refer to the documentation on preparing your data for training. 😊📁
